packages/firestore/test/integration/api/cursor.test.ts

```diff
@@ -26,7 +26,8 @@ import {
   withTestDbs
 } from '../util/helpers';
 
-const Timestamp = firebase.firestore.Timestamp;
+const Timestamp = firebase.firestore!.Timestamp;
```

Comment:
Is something wrong with our typings? Would our end users have to do something like this to enable strict null checks?

If not why do we need to do this?

Reply:
I asked Josh about this yesterday. The way our types are defined allows us to extend the `firebase` namespace after the fact without having to add all other product's APIs as well. Note that we don't ship generated typings, but only the hand-curated types in `firestore-types` and `firebase-types`.